tests/cross_validation/IMRPhenomXHM/test_lal_xhm_amplitude.py

```
@@ -0,0 +1,114 @@
+    # Convert geometric amplitude to LAL's distance-and-mass units.
+    # LAL's SimIMRPhenomXHMAmplitude returns the *physical* amplitude in
+    # SI: Amp_lm = amp0 * Amp_geom_lm with amp0 = M^2 * MRSUN * MTSUN / d.
+    from ripplegw.constants import MTSUN, MRSUN, MPC
+
+    Mtot = params["m1"] + params["m2"]
+    amp0 = Mtot * MRSUN * Mtot * MTSUN / (params["distance_mpc"] * MPC)
+    return amp_geom * amp0
+
+
+@pytest.mark.parametrize(
+    "ell,m", NON_22_MODES, ids=[f"{l}{m}" for l, m in NON_22_MODES]
```

Ruff (0.15.15)

[error] Ambiguous variable name: `l`

(E741)